vocab_tools/__init__.py

```diff
@@ -415,29 +425,45 @@ def concept(self, term: str):
             vocabulary=vocabulary,
             history=history,
             notes=notes,
-            scopenote=self.objects(term, skosT("changeNote")),
+            sources=sources,
+            #scopenote=self.objects(term, skosT("scopeNote")),
             related=self.objects(term, skosT("related")),
             example=self.objects(term, skosT("example")),
-            changenote=self.objects(term, skosT("changeNote")),
+            #changenote=self.objects(term, skosT("changeNote")),
         )
 
-    def top_concept(self) -> VocabularyConcept:
-        """Get the root concept of the specified vocabulary.
-
-        This is the concept that is skos:topConceptOf and has no skos:broader
+    def top_concept(self):
```

Comment:
Suggest adding back the typehint to assist future development.

Also should probably rename method to plural form.

Reply:
I changed it to top_concepts. 
is the typehint "-> typing.List["VocabularyConcept"]'?   I moved it into the comment while debugging because I am unclear if that's the correct type.